### ISSUE TITLE
False positive errors in CSS bindings

### ISSUE THREAD

**Issue by justinfagnani** [runem/lit-analyzer#40]
A binding like this:

```js
html`
  <style>
    :host {
      transform-origin: ${x}% ${y}%;
    }
  </style>
`;
```

Gives this error on the semi-colon on the `transform-origin` line:

```
term expectedlit-plugin(2322)
```

**Comment by justinfagnani** [comment]
Thanks @runem!!!

How fast are you getting our releases? This is blocking me from using the CLI :)

**Comment by runem** [comment]
No problems! I just released version `1.1.8` with the fix :-)
